extension/src/extension.ts

```diff
@@ -59,54 +61,42 @@ export function activate(context: vscode.ExtensionContext) {
         });
     });
 
-    const getCommandToRunSpy = (serverPath) => {
-        const os = execSync("uname -s").toString().toLowerCase();
-        if (os.indexOf("darwin") !== -1) {
-            return `osascript -e 'tell application "Terminal" to do script "node ${serverPath}"'`;
-        } else if (os.indexOf("linux") !== -1) {
-            return `xterm -e node ${serverPath}`;
+    const getServerUrl = () => {
```

Comment:
Do not put every method in the extension.ts file.
The methods related to the spyweb feature should be in another module